bats/tests/containers/factory-reset.bats

```diff
@@ -156,15 +156,15 @@ rdctl_factory_reset() {
 
 check_directories() {
     # Check if all expected directories are created after starting application/ are deleted after a factory reset
-    delete_dir=("$PATH_APP_HOME" "$PATH_CONFIG")
+    delete_dir=("$PATH_CONFIG")
     if is_unix; then
         delete_dir+=("$HOME/.rd")
         if is_macos; then
             # LIMA_HOME is under PATH_APP_HOME
             delete_dir+=("$PATH_LOGS")
         elif is_linux; then
             # Both PATH_LOGS and LIMA_HOME are under PATH_DATA
-            delete_dir+=("$PATH_DATA")
+            delete_dir+=("$PATH_DATA/logs" "$PATH_DATA/lima")
```

Comment:
For Linux, this isn't quite what we want. As it is, the factory reset code will always delete both of these paths:

- `~/.local/share/rancher-desktop/logs`
- `~/.local/share/rancher-desktop/lima`

It will not ever delete `~/.local/share/rancher-desktop/snapshots`, by design.

However, it should delete `~/.local/share/rancher-desktop` if `~/.local/share/rancher-desktop/snapshots` is empty, or not present. The code doesn't currently do this.